src/backend/jag-efiling-api/src/test/java/ca/bc/gov/open/jagefilingapi/submission/SubmissionApiDelegateImplTest.java

```diff
@@ -46,10 +46,10 @@
     private static final String SUBTYPE = "SUBTYPE";
     private static final String URL = "http://doc.com";
     private static final String HEADER = "HEADER";
-    public static final UUID CASE_4 = UUID.fromString("77da92db-0791-491e-8c58-1a969e67d2fe");
-    public static final UUID CASE_5 = UUID.fromString("77da92db-0791-491e-8c58-1a969e67d2fa");
-    public static final UUID CASE_6 = UUID.fromString("77da92db-0791-491e-8c58-1a969e67d2fb");
-    public static final UUID CASE_7 = UUID.fromString("77da92db-0791-491e-8c58-1a969e67d2fc");
+    public static final UUID CASE_6 = UUID.fromString("77da92db-0791-491e-8c58-1a969e67d2fe");
+    public static final UUID CASE_7 = UUID.fromString("77da92db-0791-491e-8c58-1a969e67d2fa");
+    public static final UUID CASE_8 = UUID.fromString("77da92db-0791-491e-8c58-1a969e67d2fb");
+    public static final UUID CASE_9 = UUID.fromString("77da92db-0791-491e-8c58-1a969e67d2fc");
```

Comment:
are these uuids supposed to be different or the same?

Reply:
No clue they are existing but I assume they should be different as they used in separate test cases.

Reply:
Was curious because they're exact down to the very last char. Just looked weird.

Reply:
yeah they are for test cases